### DISCUSSION TITLE
DiaFlux not accessible

### DISCUSSION THREAD

**Discussion by balancebot1** [denkbares/d3web-KnowWE#9] [Q&A] [answered]
With KnowWe Version 12.7 and JAVA JDK 17.0.2 DiaFlux is not accessible. 
Can someone help, please?
![Bildschirmfoto 2022-04-02 um 12 04 02](https://user-images.githubusercontent.com/17645137/161378804-ac7f905f-bef3-464f-902b-466405da2fb2.png)

**Comment by AlbrechtStriffler** [comment]
Hi, we are planning to move to Java 17 soon, but it might be a few more weeks or month. At the moment we are using Java 14 for d3web-KnowWE. Can you just switch to that?
Make sure your JAVA_HOME points to a Java 14 JDK.

**Reply by balancebot1** [reply]
Hi. Unfortunately, it is the same error message with version 14.0.2. JAVA_HOME pointing to Java 14 JDK.
Might there be any other issue?

**Comment by AlbrechtStriffler** [comment]
Which version of KnowWE are you using exactly? And which operating system? I can try and have a closer look...

**Reply by balancebot1** [reply]
Know We 12.7, 2022-02-11 16:02
Windows 11 Home 21H2

**Comment by AlbrechtStriffler** [comment, marked as answer]
Ok, finally had time for a closer look - unfortunately there was an incompatibility in DiaFlux since we transitions to a new JSPWiki build shortly before the 12.7 release. It is fixed in the nightly build and I also fixed it in the 12.7 release, but you have to download the release again.
Let me know if the new build works for you. And thanks for reporting!
